src/train.py

```diff
@@ -59,11 +57,17 @@
     # Create Result Directory if not existed
     create_directory(model_dir)
 
+    data_tiled_client = from_uri(
+        io_parameters.data_tiled_uri, api_key=io_parameters.data_tiled_api_key
+    )
+    mask_tiled_client = None
+    if io_parameters.mask_tiled_uri:
+        mask_tiled_client = from_uri(
+            io_parameters.mask_tiled_uri, api_key=io_parameters.mask_tiled_api_key
+        )
     dataset = TiledDataset(
-        data_tiled_uri=io_parameters.data_tiled_uri,
-        data_tiled_api_key=io_parameters.data_tiled_api_key,
-        mask_tiled_uri=io_parameters.mask_tiled_uri,
-        mask_tiled_api_key=io_parameters.mask_tiled_api_key,
+        data_tile_client=data_tiled_client,
```

Comment:
Minor fix of typo here.

```suggestion
        data_tiled_client=data_tiled_client,
```

Reply:
fixed